partners.yml

```diff
@@ -447,6 +447,12 @@ components:
           minimum: 0
           exclusiveMinimum: true
           example: 24
+        preferredInterestRate:
+          type: number
+          description: Preferred interest rate for the customer in percent. This can be used in case of a special partnership with Moank where the comission of the partner will depends of the interest rate sold to the end customer.
```

Comment:
I'm not sure there is an "s" at "depends" 🤔

Reply:
no it is not supposed to be there ^^